Sources/Build/BuildPlan.swift

```diff
@@ -239,7 +250,7 @@ public final class ClangTargetBuildDescription {
 
     /// Path to the bundle generated for this module (if any).
     var bundlePath: AbsolutePath? {
-        buildParameters.bundlePath(for: target)
+        target.underlyingTarget.bundleName.map(buildParameters.bundlePath(named:))
```

Comment:
should we add `target.bundleName` pointing to `target.underlyingTarget.bundleName`?

Reply:
see above